deployment/ansible/group_vars/development

```diff
@@ -15,3 +15,7 @@ postgresql_host: "{{ services_ip }}"
 relp_host: "{{ services_ip }}"
 graphite_host: "{{ services_ip }}"
 statsite_host: "{{ services_ip }}"
+
+sqs_queue: "raster-foundry-{{ host_user }}-{{ host_hostname }}"
```

Comment:
I'm just worried about filtering out characters that aren't valid for queue names. We'd have to make sure to avoid that situation if we keep it around.